app/posts/manage-teacher-training-applications/2020-08-06-diversity-information.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Showing equality and diversity information to providers
+description: How we plan to show equality information to those who should see it, but make sure it's protected
+date: 2020-08-06
+related:
+  items:
+  - text: Equality monitoring
+    href: /apply-for-teacher-training/equality-monitoring/
+  - text: Apply – September 2019
+    href: /apply-for-teacher-training/apply-september-2019/#equality-and-diversity
+---
+
+{% from "image/macro.njk" import appImage with context %}
+
+In March we started collecting equality and diversity information from candidates. This is an optional questionnaire that we ask candidates to complete to reduce discrimination on the basis of sex, disability and ethnicity.
+
+We'll only make this information available to providers once an application is complete.
+
+We're also introducing a new permission to provider accounts so that only those with ’view diversity information’ permission will be able to view the results.
+
+This means that for most people looking at most applications, they won't be able to see the diversity information.
+
+## Different display scenarios
+
+There's three new pieces of data to display, but a few different scenarios to cover.
+
+They are:
+* The candidate has completed the diversity questionnaire or not
+* The application is complete or not
+* The viewing user has permission to see diversity information or not
+
+We want providers to be clear what information is available, but also why they can't necessarily see it.
+
+## First iteration
+
+We explored adding additional rows to the applicant details summary list, but with a placeholder line if they couldn't be shown.
+
+{{ appImage({
+  file: 'multiple-lines-short-grey.png',
+  alt: 'a foo'
+})}}
+
+Once you add multiple rows it gets somewhat repetitive. We tried making them grey to knock them back a bit - and because they're more of a message from the system than a candidate's answer
```

Comment:
```suggestion
Once you add multiple rows it gets somewhat repetitive. We tried making them grey to knock them back a bit - and because they’re more of a message from the system than a candidate’s answer
```